test/integration/index-test.js

```diff
@@ -29,7 +29,7 @@ const cliUpdateCommandModule = require('../../src/index');
 const getStartAndEndModule = require('../../src/get-start-and-end-commands');
 
 describe(function() {
-  this.timeout(30e3);
+  this.timeout(240e3);
```

Comment:
Seems like there's something wrong with a timeout like this. What happened?

Reply:
Try installing ember-cli@2.11.1 globally. The tests will use it instead of downloading each time.

Reply:
Since this test is the only one requiring a full app install (since it needs the ember bin for invoking generate), lets move this test to its own describe so it only has the long timeout.

Reply:
Moved under a describe in the same file